wordpress-mobile/WordPress-Android
# RI: Init "Choose Your Interests"

Master issue: #12027

This PR introduces new fragment, VM and VMTest for Reader - "Choose Your Interests" screen. 

**To test**

0. Switch to build variant where feature flag `READER_IMPROVEMENTS_PHASE_2` is enabled
1. Go to Reader tab
2. Blank screen is displayed.

PR submission checklist:
